build.py

```diff
@@ -1281,6 +1232,58 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
     return df
 
 
+def add_cpu_libs_to_linux_dockerfile(dockerfile, backends, target_machine):
```

Comment:
can we make this part of the `BuildScript` class?

Reply:
You're right. I think what you previously had made more sense. I thought `dockerfile_prepare_container_linux` is part of the `BuildScript`.

I don't think the current PR would work since you can only call `add_cpu_libs_to_linux_dockerfile` using an instance of build script.

Reply:
Got it. I hadn't run CI with the updates yet, because I wanted to get your thoughts first. It had previously passed. Reverted the commits to get the PR back to the previous state.